HeterogeneousCore/SonicTriton/src/TritonClient.cc

```diff
@@ -91,7 +91,14 @@ TritonClient::TritonClient(const edm::ParameterSet& params)
     auto [curr_itr, success] = input_.emplace(
         std::piecewise_construct, std::forward_as_tuple(iname), std::forward_as_tuple(iname, nicInput, noBatch_));
     auto& curr_input = curr_itr->second;
-    curr_input.setConverterParams(converterDefs);
+    bool foundConverter = false;
+    for (const auto converterDef : inputConverterDefs) {

```

Comment:
This is essentially O(N^2) in the number of inputs N. Even though N is usually small, that is not ideal. It would be better to convert the initial VPSet into `unordered_map<string,string>` (inserting and searching both O(1) on average). (and do the same for the output section below)